pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/db/Tissue.java

```diff
@@ -253,18 +253,19 @@ public static String createNewTissue(@NonNull String oncHistoryId, @NonNull Stri
         SimpleResult results = inTransaction((conn) -> {
             SimpleResult dbVals = new SimpleResult();
             try (PreparedStatement stmt = conn.prepareStatement(SQL_INSERT_TISSUE, Statement.RETURN_GENERATED_KEYS)) {
-                stmt.setString(1, oncHistoryId);
```

Comment:
We seem to have an antipattern of implicit conversions.  As I see these, I am trying to fix them case-by-case.